frontend/src/components/ActivityAssessmentList/ActivityListItem.js

```diff
@@ -0,0 +1,50 @@
+import { Row } from 'react-bootstrap';
+import styled from 'styled-components';
+import { getActivityTypeName, getActivityImg } from '../../utils/constants';
+import { ActivityImg } from '../ActivityInfo/ActivityInfoStyles';
+import { useNavigate } from 'react-router-dom';
+import { PageRoutes } from '../../utils/constants';
+
+const ActivityListItemRow = styled(Row)`
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    width: 75%;
+    background-color: var(--dark-blue);
+    color: var(--font-color);
+    padding: 8px;
+    padding-right: 10px;
+    text-align: center;
+
+    &:hover {
+        cursor: pointer;
+    }
+`;
+
+export default function ActivityListItem(props) {
+    const navigate = useNavigate();
+
+    return (
+        <ActivityListItemRow
+            key={props.activity.name}
+            onClick={() => {
+                console.log('Elo od ' + props.activity.name);
+                navigate(`${PageRoutes.ACTIVITY_ASSESSMENT}`, {
+                    state: { activityId: props.activity.activityId },
+                });
+            }}
+        >
+            <ActivityImg src={getActivityImg(props.activity.type)}></ActivityImg>
+            <div>
+                {'[' +
+                    props.activity.source +
+                    '] ' +
+                    getActivityTypeName(props.activity.type) +
+                    ' - ' +
+                    props.activity.name}
```

Comment:
Nie sprawdzałem czy w tym miejscu tak można, ale ja bym to zapisał tak:
```jsx
{
    `[${props.activity.source}] ${getActivityTypeName(props.activity.type)} - ${props.activity.name}`
}
```

Ale to drobnostka i kwestia gustu w sumie, a w dodatku nie wiem czy to zadziała tutaj. Także traktuj jako drobną sugestię

Reply:
Działa, podoba mi się więc podstawiłem